pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Consumer.java

```diff
@@ -1088,7 +1088,7 @@ public Subscription getSubscription() {
 
     private int addAndGetUnAckedMsgs(Consumer consumer, int ackedMessages) {
         int unackedMsgs = 0;
-        if (Subscription.isIndividualAckMode(subType)) {
+        if (Subscription.isIndividualAckMode(subType) && subscription instanceof PersistentSubscription) {
```

Comment:
`subscription instanceof PersistentSubscription` or `isDurable` more better ?

Reply:
> `subscription instanceof PersistentSubscription` or `isDurable` more better ?

`isDurable` is better.